modules/mysql/manifests/db.pp

```diff
@@ -13,14 +13,14 @@
 )
```

Comment:
I would default charset and collate to utf8 and ut8_general_ci

Reply:
Good suggestion, thank you!